dll/cpl/sysdm/userprofile.c

```diff
@@ -645,6 +645,7 @@ UpdateButtonState(
         iSelected = ListView_GetNextItem(hwndListView, -1, LVNI_SELECTED);
         if (iSelected != -1)
         {
+            ZeroMemory(&Item, sizeof(LVITEM));
             Item.mask = LVIF_PARAM;
             Item.iItem = iSelected;
             Item.iSubItem = 0;
```

Comment:
This line becomes redundant.

Reply:
Why redundant ? It allows the proper initialization to 0 of all fields but the ones used for search. Am I wrong ?

Reply:
Oups, my mistake.
Indeed "Item.iSubItem = 0;" is useless after a ZeroMemory. This code pattern is used 4 times in this file (2 times already like this prior to the change).
Would you suggest that I remove the line setting iSubItem to 0 all 4 times ?

Reply:
Oups, my mistake.
Indeed "Item.iSubItem = 0;" is useless after a ZeroMemory. This code pattern is used 4 times in this file (2 times already like this prior to the change).
Would you suggest that I remove the line setting iSubItem to 0 all 4 times ?